studiopress/genesis-portfolio-pro
# Remove Grunt and add pot generation using wp-cli

**Summary of change:**
This PR removes Grunt and add support generating the `.pot` file using wp-cli.

**This PR has been:**

**Have the changes in this PR been added to the documentation for this project?**
No

**How to test:**
- Download this branch;
- Run `composer install`;
- Generate the `.pot` file by running `composer makepot`;
- Check if the `.pot` file has been properly generated.
